propolis/src/hw/nvme/mod.rs

```diff
@@ -1,5 +1,6 @@
 use std::collections::BTreeMap;
 use std::convert::TryInto;
+use std::mem;
```

Comment:
style nit: maybe just import `size_of` directly, since that's the only thing being used (and is unambiguous in the file)